public/js/jquery.t.js

```diff
@@ -340,7 +357,7 @@
             }
             // Bind listener events once all the graphical elements have
             // been drawn for first time.
-            if (index === 0) {
+            if (index === 0 && resize !== 1 ) {
```

Comment:
Can you document the `resize` parameter (as in, why it is important)? May be in `setupResponsiveness` or here itself.